### PR TITLE
Add tutorial that shows how to set up Prefect Cloud as a platform for data teams

### PR DESCRIPTION
**Pull request by daniel-prefect**
Closes [DOC-95](https://linear.app/prefect/issue/DOC-95/create-an-operator-tutorial-for-setting-up-a-data-platform)

Preview: https://prefect-bd373955-operator_tutorials.mintlify.app/v3/tutorials/platform

This tutorial [depends on a PR](https://github.com/PrefectHQ/demos/pull/1) with a script which generates failing flow runs on demand.

### Checklist

- [X] This pull request references any related issue by including "closes `<link to issue>`"
- [ ] ~If this pull request adds new functionality, it includes unit tests that cover the changes~
- [ ] ~If this pull request removes docs files, it includes redirect settings in `mint.json`.~
- [ ] ~If this pull request adds functions or classes, it includes helpful docstrings.~

### CODE REVIEW

File: docs/v3/tutorials/platform.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+- The script created two deployments of the flow, one deployment per workspace.
+- The script then generated multiple flow runs for each deployment.
+- The activity graph on the **Home** page shows successful runs as green and failed runs as red.
+- Notice that the flow runs in the `production` workspace succeeded, but some of the flow runs in the `staging` workspace failed.
+
+This output is expected, you'll learn how to debug these failures in the next tutorial.
+
+## Optional: Manage team access
+
+If you're a Prefect Enterprise customer, you can use Teams to manage access to your workspaces and work pools.
```
Comment (discdiver) [inline docs/v3/tutorials/platform.mdx:137]: Consider linking to teams docs page/section here or below.

**Review by discdiver — COMMENTED** [review]
Looking good. A few minor comments.

File: docs/v3/tutorials/platform.mdx (hunk 1 above)
Comment (daniel-prefect) [reply docs/v3/tutorials/platform.mdx]: https://github.com/PrefectHQ/prefect/pull/16123/commits/f38f3782236ea0a3526e78ddb8cb44d6834c2524

File: docs/v3/tutorials/platform.mdx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+## Start workers
+
+Because you're using hybrid work pools, you need to start at least one worker for each work pool to process flow runs.
+
+First, you need to authenticate to Prefect Cloud using the CLI.
+
+```bash
+prefect cloud login
+```
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/platform.mdx:74]: https://github.com/PrefectHQ/prefect/pull/16123/commits/6c9fc2e95560b6098b6e61d323135721e67cef75

**Comment by daniel-prefect** [comment]
Thanks for the review @discdiver. I think I've addressed all of your feedback. 👍

**Review by discdiver — APPROVED** [review]
LGTM. FWIW, our other screenshots of the UI are all in dark mode if you want to match.

**Comment by daniel-prefect** [comment]
> LGTM. FWIW, our other screenshots of the UI are all in dark mode if you want to match.

I got you. https://github.com/PrefectHQ/prefect/pull/16123/commits/ac8398ed2ac1864649113b221e9c564f31c818d5

**Pull request merged**